src/test/java/io/github/eliux/mega/cmd/ImportInfoTest.java

```diff
@@ -0,0 +1,105 @@
+package io.github.eliux.mega.cmd;
+
+import io.github.eliux.mega.Mega;
+import io.github.eliux.mega.MegaSession;
+import io.github.eliux.mega.error.MegaInvalidResponseException;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ImportInfoTest {
+
+  private static MegaSession sessionMega;
+
+  @BeforeClass
+  public static void setupSession() {
+    sessionMega = Mega.init();
+  }
+
+  @AfterClass
+  public static void finishSession() {
+    //sessionMega.logout();
```

Comment:
If this is not needed better remove function, but it is adviced that after you start a session you logout from it: relevant when running just one test file.

Reply:
This is my first contribution, this was a mistake.

Reply:
It actually was a very good start. This are just small details.